strings.txt

```diff
@@ -2377,7 +2377,7 @@
   [add_new_set]
     comment = Add New Bookmark Set dialog title
     tags = ios,tizen
-    en = Add new Set
+    en = Add New Set
```

Comment:
It's ok to use it without the article for brevity:
![screen shot 2016-08-09 at 19 39 36](https://cloud.githubusercontent.com/assets/14867234/17524963/b77a01ec-5e71-11e6-88cb-8d56ef554bda.png)